scripts/generateFiles.ts

```diff
@@ -195,7 +195,16 @@ module LanguageServer
       {{/if}}
       class {{definition.interface.name}}{{#if definition.parent}} < {{definition.parent.interface.name}}{{/if}}
         def initialize({{params definition.allMembers}})
+          {{#if definition.parent}}
+          super(
+            {{#each definition.parent.members}}
+            {{name}}: {{local_var name}},
+            {{/each}}
+          )
+          @attributes = @attributes.dup
+          {{else}}
           @attributes = {}
+          {{/if}}
 
           {{#each definition.members}}
```

Comment:
Rewriting this `definition.members` to `definition.allMembers` would be a simpler solution for this...

Reply:
I think it should be `definition.allMembers` :bow: